spec/support/messaging/cy.yml

```diff
@@ -0,0 +1,313 @@
+et3-v1-cy:
+  response_pdf_fields:
+    header:
+      case_number:
+        field_name: 'Rhif yr'
+      date_received:
+        field_name: false
+      rtf:
+        field_name: false
+    claimant:
+      claimants_name:
+        field_name: 'undefined'
```

Comment:
Yes, we truly do have fields called 'undefined' in the PDF - not under our control though